python/foglamp/services/core/server.py

```diff
@@ -235,6 +243,8 @@ def get_certificates():
             await cls._configuration_manager.create_category(category, config, 'The FogLAMP Admin and User REST API', True)
             config = await cls._configuration_manager.get_category_all_items(category)
 
+            cls.cert_file_name = config['certificateName']['value']
```

Comment:
move into try except block

except KeyError:
                 _logger.error("error in retrieving certificate name")
raise